spec/rubocop/cop/bundler/ordered_gems_spec.rb

```diff
@@ -131,17 +133,52 @@
     end
   end
 
-  context 'When gem groups is separated by comment' do
+  context 'When gem groups is separated by multiline comment' do
     let(:source) { <<-END }
       # For code quality
       gem 'rubocop'
-      # For test
+      # For
+      # test
       gem 'rspec'
     END
 
-    it 'accepts' do
-      inspect_source(cop, source)
-      expect(cop.offenses).to be_empty
+    context 'with enforced style set to `comment_lines_split`' do
+      let(:enforced_style) { 'comment_lines_split' }
+
+      it 'accepts' do
+        inspect_source(cop, source)
+        expect(cop.offenses).to be_empty
+      end
+    end
+
+    context 'with enforced style set to `no_comment_lines_split`' do
+      it 'registers an offense' do
+        inspect_source(cop, source)
+        expect(cop.offenses.size).to eq(1)
+      end
+
+      it 'has the correct offense message' do
+        inspect_source(cop, source)
+        expect(cop.messages)
+          .to eq(['Gems should be sorted in an alphabetical '\
+                'order within their section of the Gemfile. '\
```

Comment:
This should be aligned with the ' on the previous line.

cc @jonas054 - another cop alignment idea :-)